src/lib/dataEntities/networks.ts

```diff
@@ -28,14 +28,16 @@ export interface L1Network extends Network {
 }
 
 export interface L2Network extends Network {
+  partnerChainIDs: number[]
```

Comment:
This will actually be a breaking change, so we should make it optional probably, and let's add jsdoc

We can also then remove the changes of adding this to all the networks